AwsCryptographicMaterialProviders/runtimes/python/README.rst

```diff
@@ -0,0 +1,78 @@
+####################################################
+aws-cryptographic-materials-providers-library-python
+####################################################
+
+.. image:: https://github.com/aws/aws-cryptographic-material-providers-library/actions/workflows/push.yml/badge.svg
+   :target: https://github.com/aws/aws-cryptographic-material-providers-library/actions/workflows/push.yml
+   :alt: MPL-python-tests
+
+.. image:: https://img.shields.io/badge/code_style-black-000000.svg
+   :target: https://github.com/ambv/black
+   :alt: Code style: black
+
+.. image:: https://readthedocs.org/projects/aws-cryptographic-material-providers-library/badge/
+   :target: https://aws-cryptographic-material-providers-library.readthedocs.io/en/latest/
+   :alt: Documentation Status
+
+These are the docs for the AWS Cryptographic Materials Providers Library for Python.
+
+The latest full documentation can be found at `Read the Docs`_.
+
+Find us on `GitHub`_.
+
+Security
+********
+
+If you discover a potential security issue in this project
+we ask that you notify AWS/Amazon Security via our
+`vulnerability reporting page`_.
+Please **do not** create a public GitHub issue.
+
+***************
+Getting Started
+***************
+Required Prerequisites
+======================
+
+* Python 3.11+
+* cryptography >= 41.0.0
+* boto3 >= 1.28.0
+
+Installation
+============
+
+.. note::
+
+   If you have not already installed `cryptography`_, you might need to install additional prerequisites as
+   detailed in the `cryptography installation guide`_ for your operating system.
+
+   .. code::
+
+       $ pip install aws-cryptographic-material-providers
+
+
+Concepts
+========
+The AWS Cryptographic Materials Providers Library (MPL) is available in multiple languages.
+The concepts in the Python implementation of the MPL are the same as in other languages.
+For more information on concepts in the MPL, see the `README`_ for all languages.
+
+*************
+Thread safety
+*************
+
+The ``MaterialProviders`` client and all provided ``CryptoMaterialsManagers`` (CMMs) are thread safe.
+But keyrings that call KeyManagementService (KMS) and CMMs that use keyrings that call KMS MUST not be shared between threads, for reasons outlined in `the boto3 docs`_.
```

Comment:
nit: maybe AWS Key Management Service (KMS) for the first mention, KMS alone is underspecified